components/social-sign-up.tsx

```diff
@@ -44,6 +58,7 @@ const SocialSignUp = ({ title = "signup" }: { title?: string }) => {
             height={18}
           />,
           "Slack",
+          "https://auth-leave-tracker.stixor.com/v1/auth/slack/login",
```

Comment:
Remove these urls from here, keep them in constants and use here instead of directly pasting the base url here